build/tutorials-contest/liquid-avax/Backend/backend.js

```diff
@@ -0,0 +1,272 @@
+// This script will be used to do the cross-chain transfer of the owner wallet from C to P chain and stake them
+// to the user-chosen node.
+
+import fs from "fs";
+import {Avalanche, BinTools, BN} from "avalanche";
+import Web3 from "web3";
+import { Buffer } from "buffer";
+
+let xChain, xKeyChain, xChainAddress;
+let cChain, cKeyChain, cChainAddress;
+let pChain, pKeyChain, pChainAddress;
+
+let xChainBlockchainID, cChainBlockchainID, pChainBlockchainID;
+
+let web3;
+
+let contractAbi;
+
+let masterAddress = "0x7bD7A7D2Ba70db40740780828b236F9246BB7F78";
+let privateKey = JSON.parse(await fs.readFileSync("./data.json")).privateKey;
+
+let currentTxId = [];
+
+let utxoset;
+
+
+async function waitForStaked() {
+    let stakingContract = new web3.eth.Contract(contractAbi, "0x5bD27eF83d57915FC3eE7feB2FebEB9c69d52B04");
+    let stakedEvent = stakingContract.events.Staked();
+    await web3.eth.subscribe('logs', {
+        address: stakedEvent.arguments[0].address,
+        topics: stakedEvent.arguments[0].topics,
+    }, async function (error, logs) {
```

Comment:
Use fat arrow instead of `function`. Consider `function` deprecated and refactor all instances to fat arrow.